tests/handlers/test_q30_handler.py

```diff
@@ -9,43 +9,79 @@ class TestQ30Handler(HandlerTestBase):
 
     def setUp(self):
         key = "percent_q30"
-        qc_config = {'name': 'Q30Handler', 'error': 70, 'warning': 80}
+        qc_config = {"name": "Q30Handler", "error": 70, "warning": 80}
         value_1 = {"lane": 1, "read": 1, "percent_q30": 82}
-        value_2 = {"lane": 1, "read": 2, "percent_q30": 90}
-        value_3 = {"lane": 1, "read": 3, "percent_q30": 50, "is_index_read": True}
+        value_2 = {"lane": 1, "read": 2, "percent_q30": 50, "is_index_read": True} 
+        value_3 = {"lane": 1, "read": 3, "percent_q30": 50, "is_index_read": True} 
+        value_4 = {"lane": 1, "read": 4, "percent_q30": 60}
+        value_5 = {"lane": 2, "read": 1, "percent_q30": 90}
+        value_6 = {"lane": 2, "read": 2, "percent_q30": 50, "is_index_read": True}
+        value_7 = {"lane": 2, "read": 3, "percent_q30": 50, "is_index_read": False}
+        value_8 = {"lane": 2, "read": 4, "percent_q30": 40}
         q30_handler = Q30Handler(qc_config)
         q30_handler.collect((key, value_1))
         q30_handler.collect((key, value_2))
         q30_handler.collect((key, value_3))
+        q30_handler.collect((key, value_4))
+        q30_handler.collect((key, value_5))
+        q30_handler.collect((key, value_6))
+        q30_handler.collect((key, value_7))
+        q30_handler.collect((key, value_8))
         self.q30_handler = q30_handler
 
     def set_qc_config(self, qc_config):
         self.q30_handler.qc_config = qc_config
 
     def test_all_is_fine(self):
-        qc_config = {'name': 'Q30Handler', 'error': 70, 'warning': 80}
+        qc_config = {"name": "Q30Handler", "error": 70, "warning": 80}
         self.set_qc_config(qc_config)
         errors_and_warnings = list(self.q30_handler.check_qc())
         self.assertIn("index read", f"{errors_and_warnings}")
-        self.assertEqual(f"{errors_and_warnings}", "[Fatal QC error: %Q30 50.00 was too low on lane: 1 for index read: 1]")
+        self.assertEqual(
+            f"{errors_and_warnings}",
+            "[Fatal QC error: %Q30 50.00 was too low on lane: 1 for index read: 1, Fatal QC error: %Q30 50.00 was too low on lane: 1 for index read: 2, Fatal QC error: %Q30 60.00 was too low on lane: 1 for read: 2, Fatal QC error: %Q30 50.00 was too low on lane: 2 for index read: 1, Fatal QC error: %Q30 50.00 was too low on lane: 2 for read: 2, Fatal QC error: %Q30 40.00 was too low on lane: 2 for read: 3]",
```

Comment:
I realize I should have reacted to this during your last PR, but I guess I missed it: The purpose of this test was originally to test the case where the handler does not report any warnings or errors (hence the name all_is_fine). See         [self.assertEqual(errors_and_warnings, [])](https://github.com/Molmed/checkQC/blob/639d3e2c0555ae3b4810fdef542fe22ba48ade0c/tests/handlers/test_q30_handler.py#L27). I think the qc_config for this case should be updated so that is the case, for example: 
```
qc_config = {"name": "Q30Handler", "error": 20, "warning": 30}
```

It would also be fine to restructure the tests, I think the most important thing is that the test name reflect what we are testing.